modules/install-consul/install-consul

```diff
@@ -300,7 +300,7 @@ function install {
     install_tls_certificates "$path" "$user" "$ca_file_path" "$cert_file_path" "$key_file_path"
   fi
 
-  if command -v consul; then
+  if sudo bash -c "command -v consul"; then
```

Comment:
NIT: could you add a comment above this line explaining why we need `sudo` and linking back to #210?

Reply:
Done!